src/data/message_formats.py

```diff
@@ -21,72 +24,85 @@ def format(
         key_object_info: dict,
         image_path: str,
         system_prompt: str,
-        answer: str="",
+        answer: Optional[str] = None,
+        min_pixels: Optional[int] = None,
+        max_pixels: Optional[int] = None,
     ) -> dict[str, str | list[dict[str, str]]]:
+        image_dict = {
+            "type": "image",
+            "image": f"file://{image_path}",
+        }
+        if min_pixels is not None:
+            image_dict["min_pixels"] = min_pixels
```

Comment:
This wasnt clear to me before, but looking at the doc link you posted, the setting in the model and the setting in the message format seem to be doing the same thing. I.e. one of them is unnecessary. Seeing that we will have to do this for every message i would opt for leaving the message format untouched and just routing it through the model where the max_pixels are already set.